style.css

```diff
@@ -0,0 +1,142 @@
+html {
+    font-size: 10px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+}
+
+h1 {
+    color: #ffffff;
+    font-weight: 700;
+    font-size: 10rem;
+    text-align: center;
+}
+
+h2 {
+    font-size: 7rem;
+}
+
+p {
+    font-size: 1.8rem;
+}
+
+.nav-bar {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    color: #ff8000;
+    font-size: 1.2rem;
+    text-transform: uppercase;
+    font-weight: 400;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+    color: #ff8000;
+    text-decoration: none;
+}
+
+.nav-bar a:hover {
+    color: #001433;
+    text-decoration: underline;
+}
+
+.nav-menu {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+
+}
+
+.nav-menu-item {
+    margin: 0.2em;
+    padding: 0.6em;
+}
+
+.nav-title {
+    font-size: 2.5rem;
+    font-weight: 700;
+    letter-spacing: 0.15rem;
+}
+
+header {
+    background-color: #ff8000;
+    padding-top: 25px;
+    padding-bottom: 25px;
+    min-height: 60vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.intro-section {
+    padding-top: 25px;
+    padding-bottom: 25px;
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 50px;
+    color: #001433;
+}
+
+.intro-section div {
+    width: 300px;
+}
+
+.intro-section-message {
+    padding-top: 30px;
+    padding-bottom: 30px;
+}
+
+.intro-section-image {
+    max-width:100%;
+    max-height:100%;
+}
+
+hr {
+    border: 1px solid #DDDDDD
+}
+
+.timeline-section {
+    background-color: #ff8000;
+    color: #ffffff;
+    padding-top: 25px;
+    padding-bottom: 25px;
+    display: flex;
+    flex-flow: column wrap;
+    align-items: center;
+    justify-content: center;
+}
+.timeline-parent {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: center;
+    width: 50%;
```

Comment:
better to have instead `max-width: 800px` or something like that, plus some margin - that way the element is allowed to shrink down on mobile but not able to grow to full width on desktop. the margin then makes sure that it doesn't touch the sides on mobile, so you don't need to have a media query at all. it might be a good idea to make sure that the flex rows align as you want them on mobile (the year with the respective thing that happened)

(as discussed)